cmakeConsole/.vscode/tasks.json

```diff
@@ -35,7 +35,7 @@
                 "-DCMAKE_BUILD_TYPE=Debug",
                 "-DCMAKE_CXX_COMPILER=arm-linux-gnueabihf-g++",
                 "-DCMAKE_C_COMPILER=arm-linux-gnueabihf-gcc",
-                "-Bbuild-armhf"
+                "-Bbuild-arm"
```

Comment:
so, this is wrong also on the `bookworm` branch?

Reply:
It is not wrong actually, it is just that if we use `build-arm` instead of `build-armhf` we can pass the build directory like this on the `COPY build-${IMAGE_ARCH}/debug ${APP_ROOT}/app`